src/test/java/winstone/HttpConnectorFactoryTest.java

```diff
@@ -22,6 +22,7 @@ public void testListenAddress() throws Exception {
         args.put("warfile", "target/test-classes/test.war");
         args.put("prefix", "/");
         args.put("httpPort", "0");
+        // see https://superuser.com/a/458877/714797 for getting this to work on macOS
```

Comment:
`just see README development part` as it's already there.
no need to link to an external website which has exactly the same content as a file already here

Reply:
```suggestion
        // see README development section for getting this to work on macOS
```